### PR TITLE
Updated metadata README.

### PR DESCRIPTION
**Pull request by mehrdadhalali**
MEHRBRANCH RETURNS

### CODE REVIEW

File: short_term/test_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi
```
Comment (Leon-JS02) [inline short_term/test_file.txt:1]: ???

File: collect_metadata/README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
-This folder contains scripts that were used to gather metadata from the API, to seed the database. You do not need to interact with this folder.
+## 🛠️ Prerequisites
+Before you start, ensure that you have the following configured on your machine:
+- **Python** to run the scripts.
 
 
+## 📂 Setup
+Follow these steps to deploy the infrastructure:
 
+1. Create an environment and run `pip install -r requirements.txt`.
```
Comment (Leon-JS02) [inline collect_metadata/README.md:12]: Very informative steps

File: short_term/test_file.txt (hunk 1 above)
Comment (YakMan101) [reply short_term/test_file.txt:1]: Nani???

Comment (mehrdadhalali) [reply short_term/test_file.txt:1]: Don't delete it, the entire project won't work without this. I don't know why.

**Review by Leon-JS02 — APPROVED** [review]
Hi.

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**